odbc/src/api/data.rs

```diff
@@ -74,7 +74,26 @@ impl IntoCData for Bson {
 
     fn to_binary(self) -> Result<Vec<u8>> {
         match self {
+            Bson::Double(f) => Ok(f.to_le_bytes().to_vec()),
+            Bson::String(s) => Ok(s.into_bytes()),
+            Bson::Boolean(b) => Ok(if b { vec![1u8] } else { vec![0u8] }),
+            Bson::DateTime(d) => {
+                let dt: DateTime<Utc> = d.into();
+                let mut ret = Vec::with_capacity(16usize);
+                ret.extend(dt.year().to_le_bytes());
+                ret.extend(dt.month().to_le_bytes());
+                ret.extend(dt.day().to_le_bytes());
+                ret.extend(dt.hour().to_le_bytes());
+                ret.extend(dt.minute().to_le_bytes());
+                ret.extend(dt.second().to_le_bytes());
+                let fraction = (dt.nanosecond() as f32 * 0.000001) as u32;
```

Comment:
Why `0.000001`?

Reply:
Fractional seconds in the Timestamp struct are 10ths of seconds rather than nanoseconds. I actually arrived at this experimentally because that number doesn't seem to correctly go from 10ths to billionths.